vmm/src/config.rs

```diff
@@ -2143,6 +2149,16 @@ impl VmConfig {
             }
         }
 
+        #[cfg(feature = "sev_snp")]
+        {
+            let host_data_opt = &self.payload.as_ref().unwrap().host_data;
+
+            if let Some(host_data) = host_data_opt {
+                if host_data.is_empty() || (!host_data.is_empty() && host_data.len() != 64) {
```

Comment:
You can simply do: `if host_data.len() != 64`

Reply:
@likebreath Updated. Thank you!